onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -566,19 +565,19 @@ def get_entropy_threshold(self, histogram, num_quantized_bins):
         from scipy.stats import entropy
         import copy
 
-        hist, hist_edges, _, _, _ = histogram
+        hist, hist_edges, min_data, _, _ = histogram
         num_bins = hist.size
         zero_bin_index = num_bins // 2
         num_half_quantized_bin = num_quantized_bins // 2
         
-        kl_divergence = np.zeros(zero_bin_index - num_half_quantized_bin + 1)
-        thresholds = [(0, 0) for i in range(kl_divergence.size)] 
+        kl_divergence = np.zeros((zero_bin_index - num_half_quantized_bin + 1) // num_half_quantized_bin + 1)
+        thresholds = [(0, 0) for i in range(kl_divergence.size)]
 
-        for i in range(num_half_quantized_bin, zero_bin_index + 1, 1):
-            start_index = zero_bin_index - i 
+        for i in range(num_half_quantized_bin, zero_bin_index + 1, num_half_quantized_bin):
```

Comment:
Could you elaborate more why you want to change here? or what document are you referencing to? 
The naive way is to increase the size of candidate distribution/bin by 1 every time until it reaches the limit.